sdcm/cluster_aws.py

```diff
@@ -824,6 +824,7 @@ def destroy(self):
         self.stop_task_threads()
         self.wait_till_tasks_threads_are_stopped()
         self._instance.terminate()
+        self._instance.wait_until_terminated()
```

Comment:
this looks like a good enough solution

Reply:
it is not enough, since listing can get you stopped nodes (or all kind of other different states)

Reply:
this one also have potential of changing nemesis duration, which might show up in performance test.

but it would be more the few minutes...